### ISSUE TITLE
[Feature Request] Rule to disallow no space between attributes

### ISSUE THREAD

**Issue by maheshbansod** [yeonjuan/html-eslint#119]
There's a rule that disallows extra space between attributes.

I'm looking for a rule that would cause an error when there is no space at all between attribute value and the next attribute.
For example, this should cause an error:
```html
<img src="image.png"alt="Image description" />
```

**Comment by maheshbansod** [comment]
I tried adding an option `disallowMissing` in the `no-extra-spacing-attrs` rule here: https://github.com/maheshbansod/html-eslint/tree/disallow-missing . It does the same as described above but instead of adding a rule, I just modified the existing rule to have an option.

Not sure if that's the way to go - seemed the easiest though. I can open a PR if needed.

Thanks,
Mahesh.

**Comment by yeonjuan** [comment]
@maheshbansod I think adding options in the `no-extra-spacing-attrs` seems fine.(IMO, Some day, the rule name should be changed). I think it would be helpful if you could make a PR. 👍

**Comment by yeonjuan** [comment]
@maheshbansod It published in `v0.16.0`. Thanks for your contribution.
